### PR TITLE
EG-4029 Added missing code examples to CE 4.4/4.5/4.6 API Transaction…

### PR DESCRIPTION
**Pull request by ivanterziev-r3**
EG-4029 Added missing code examples to CE 4.4/4.5/4.6 API Transaction pages (CorDapps / Developing CorDapps > Writing CorDapp flows > Understanding Transactions).

### CODE REVIEW

File: content/en/docs/corda-enterprise/4.4/cordapps/api-transactions.md [hunk 1]
```diff
@@ -223,6 +508,35 @@ The next step is to build up the transaction proposal by adding the desired comp
 We can add components to the builder using the `TransactionBuilder.withItems` method:
 
 {{< tabs name="tabs-14" >}}
+{{% tab name="kotlin" %}}
```
Comment (taylorharvey-r3) [inline content/en/docs/corda-enterprise/4.4/cordapps/api-transactions.md:511]: Should there be a java sample here too?

File: content/en/docs/corda-enterprise/4.4/cordapps/api-transactions.md [hunk 2]
```diff
@@ -315,6 +886,22 @@ A `SignedTransaction` is a combination of:
 * A list of signatures over that transaction
 
 {{< tabs name="tabs-26" >}}
+{{% tab name="kotlin" %}}
```
Comment (taylorharvey-r3) [inline content/en/docs/corda-enterprise/4.4/cordapps/api-transactions.md:889]: Should there be a java sample here too?

File: content/en/docs/corda-enterprise/4.5/cordapps/api-transactions.md [hunk 3]
```diff
@@ -219,6 +439,35 @@ The next step is to build up the transaction proposal by adding the desired comp
 We can add components to the builder using the `TransactionBuilder.withItems` method:
 
 {{< tabs name="tabs-14" >}}
+{{% tab name="kotlin" %}}
```
Comment (taylorharvey-r3) [inline content/en/docs/corda-enterprise/4.5/cordapps/api-transactions.md:442]: Should there be a java example here too?

File: content/en/docs/corda-enterprise/4.5/cordapps/api-transactions.md [hunk 4]
```diff
@@ -311,6 +762,22 @@ A `SignedTransaction` is a combination of:
 * A list of signatures over that transaction
 
 {{< tabs name="tabs-26" >}}
+{{% tab name="kotlin" %}}
```
Comment (taylorharvey-r3) [inline content/en/docs/corda-enterprise/4.5/cordapps/api-transactions.md:765]: Should there be a java example here too?

File: content/en/docs/corda-enterprise/4.6/cordapps/api-transactions.md [hunk 5]
```diff
@@ -311,6 +762,22 @@ A `SignedTransaction` is a combination of:
 * A list of signatures over that transaction
 
 {{< tabs name="tabs-26" >}}
+{{% tab name="kotlin" %}}
```
Comment (taylorharvey-r3) [inline content/en/docs/corda-enterprise/4.6/cordapps/api-transactions.md:765]: Should there be a java example here too?

File: content/en/docs/corda-enterprise/4.6/cordapps/api-transactions.md [hunk 6]
```diff
@@ -219,6 +439,35 @@ The next step is to build up the transaction proposal by adding the desired comp
 We can add components to the builder using the `TransactionBuilder.withItems` method:
 
 {{< tabs name="tabs-14" >}}
```
Comment (taylorharvey-r3) [inline content/en/docs/corda-enterprise/4.6/cordapps/api-transactions.md:441]: Should there be a java example here too?

**Review by taylorharvey-r3 — COMMENTED** [review]
There were 2 places (commented on in each version) where there were only kotlin examples. Should there be java examples too? Everything else looks good.

**Comment by ivanterziev-r3** [comment]
@taylorharvey-r3 Correct, some examples are Kotlin-only. However this is the original content dating back to CD 4.3, so I've added the page to https://r3-cev.atlassian.net/browse/EG-1528 and we'll tackle this issue when we add Java examples to all such pages.

**Review by taylorharvey-r3 — APPROVED** [review]
LGTM

**Pull request merged**